Tools/Scripts/libraries/reporelaypy/reporelaypy/__init__.py

```diff
@@ -23,6 +23,9 @@
 import os
 import sys
 
+if sys.version_info < (3, 6):
```

Comment:
We shouldn't need this, `reporelaypy` has never supported Python 2

Reply:
To be fair, it doesn't seem harmful to have this here—IIRC, from memory, the tests all pass with Python 3.6, but we can of course bump this higher (but I was starting with being as conservative as possible!).